pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1227,6 +1242,22 @@ func doActivate(ctx *domainContext, config types.DomainConfig,
 		return
 	}
 
+	//TODO: mountPoints must be fill with target paths on the container once we decide UI to domainmgr flow for vol mount.
```

Comment:
What is this about? What part is not clear at this point?

Reply:
Restructured it to fetch MountPoints from AppManifest